src/components/SectionEventOptions/index.js

```diff
@@ -5,7 +5,7 @@ import Checkbox from 'components/FormCheckbox'
 
 const SectionEventOptions = () => (
   <div className="sectionEventOptions">
-    {/*<Field component={Checkbox} name="ultimateOracle" label="Ultimate Oracle" text="This Event should use an ultimate Oracle." />*/}
+    {/* <Field component={Checkbox} name="ultimateOracle" label="Ultimate Oracle" text="This Event should use an ultimate Oracle." />*/}
```

Comment:
Is this component used somewhere in the app?

Reply:
@giacomolicari Probably not, this file was changed by ESlint, so i didn't take a look at it. Will check it later

Reply:
Ok, the PR is fine to me.